.github/workflows/pr-labeler.yml

```diff
@@ -27,6 +29,8 @@ jobs:
   # Changes to some paths should automatically add the changeset-required label. This label is often added manually, so
   # this CI job calls the labeler action wuth sync-labels=false, so the label won't be removed automatically.
   changesets_label:
+    permissions:
+      pull-requests: write  # for actions/labeler to add labels to PRs
```

Comment:
(commenting here since I can't comment on the later part as there's no diff)

Does `branches_label` not also require pull request write? Seems like it would.